opencti-platform/opencti-graphql/src/migrations/1722428479372-trigger-authorized-authorities.js

```diff
@@ -0,0 +1,30 @@
+import { logApp } from '../config/conf';
+import { executionContext, SYSTEM_USER } from '../utils/access';
+import { elList, elReplace } from '../database/engine';
+import { READ_INDEX_INTERNAL_OBJECTS } from '../database/utils';
+import { ENTITY_TYPE_TRIGGER } from '../modules/notification/notification-types';
+
+const message = '[MIGRATION] update authorized authorities for triggers';
+
+export const up = async (next) => {
+  logApp.info(`${message} > started`);
+  const context = executionContext('migration');
+
+  // ------ Update triggers
+  const callback = async (notifiers) => {
+    for (let i = 0; i < notifiers.length; i += 1) {
+      const notifier = notifiers[i];
+      const patch = { authorized_authorities: ['SETTINGS_SECURITYACTIVITY'] };
```

Comment:
this change should only apply on Activity Triggers, not all triggers, right ?

Reply:
I have updated this migration to only apply on activity triggers.